RecyclerView/app/src/main/java/net/boondockradio/recyclerview/dto/StateItem.java

```diff
@@ -0,0 +1,10 @@
+package net.boondockradio.recyclerview.dto;
+
+public class StateItem {
+
+    public String title;
+
+    public StateItem(String title) {
+        this.title = title;
+    }
```

Comment:
In normal java it is recommended to use `getters` and `setters`. 
**But android is different.** 
Method count can be important